resources/js/Pages/Praktikan.vue

```diff
@@ -2105,6 +2159,51 @@ export default {
           } 
         }
       });
+    },
+
+    formPassword: function($bool){
+      if($bool){
+        this.viewPassForm = true;
+      }else{
+        this.viewPassForm = false;
+        this.resetPass.password = '';
+        this.resetPass.repeatpass = '';
+      }
+    },
+
+    resetPassword: function(){
+      const globe = this;
+        this.$axios.post('/resetPass', this.resetPass).then(response => {
+
+        if(response.data.message === "success") {
+          this.$toasted.global.showSuccess({
+            message: "Password berhasil diperbaharui"   
+          });
+      
+
+          this.signOut();
+
+        } else {
+          this.$toasted.global.showError({
+            message: response.data.message
+          });
+        }
+      }).catch(function (error) {
+        if (error.response) {
+          // The request was made and the server responded with a status code
+          // that falls out of the range of 2xx
+          if(error.response.data.errors != null){
+            if(error.response.data.errors.password != null)
+              globe.$toasted.global.showError({
+                message: error.response.data.errors.password[0]
+              });
+            if(error.response.data.errors.resetPass != null)
+              globe.$toasted.global.showError({
+                message: error.response.data.errors.password[0]
```

Comment:
Im really sure there is a mistake here wkwk, i'll let you find out what it is @adhiiimf

Reply:
wkwk its okay, you can fix it now

Reply:
oh noo, not resetPass, should be resetpass wkwkw sorry